client/Hooks/useLogger/Hook.ts

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+import { useEffect, useRef } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+
+export const MESSAGE_BUFFER_MAX_SIZE = 100;
+// TODO: Replace with the regex from the LOGGING query param
+const defaultLoggingRegex = /.+/;
+// TODO: Replace with the web socket address from config
+const defaultWebSocketAddress = 'ws://localhost:3000/log';
+
+// Unique client ID created when this module is loaded
+const clientID = uuidv4();
+
+export type LogLevelType =
+  | 'fatal'
+  | 'error'
+  | 'warn'
+  | 'info'
+  | 'debug'
+  | 'trace';
+
+interface loggerMessage {
```

Comment:
Are these all required?

Reply:
These are all the levels available in the `pino` library used on the server. Only those at the `pino` logging level will actually get emitted to the log.